outputs.tf

```diff
@@ -2,6 +2,14 @@ output "public_ip" {
   value = "${aws_eip.default.public_ip}"
 }
 
+output "private_ip" {
+  value = "${aws_instance.default.private_ip}"
+}
+
+output "private_dns" {
```

Comment:
I'm torn. In the past, I've said use the original resource name (in this case `private_dns`). However, we're using `public_hostname` below. I think we should use one convention or another, but not both.

Reply:
I defer to @comeanother to pick the right convention.

Reply:
I think we should use  the original resource name. I'll rename the variable `public_hostname` to `public_dns`